src/main/resources/web_de.properties

```diff
@@ -49,4 +49,4 @@ template.html.uploadArea = Ziehen Sie Dateien hierher, um diese hochzuladen…
 template.html.uploadBtn = Datei auswählen…
 template.html.uploadFailed = Upload fehlgeschlagen
 UserInfo.missingPermission = Sie benötigen die Berechtigung "${permission}" um diese Aktion auszuführen.
-WebContext.invalidCSRFToken = Ihre Anfrage konnte nicht verifiziert werden. Bitte versuchen Sie es erneut.
+WebContext.invalidCSRFToken = Ihre Anfrage konnte nicht verifiziert werden. Bitte versuchen Sie es erneut.
```

Comment:
Bitte die leere Zeile am Ende wieder einfügen und in deinen IntelliJ Settings überprüfen ob folgendes Setting unter Editor > General aktiv ist:

![image](https://user-images.githubusercontent.com/2427877/117632175-47e2b400-b17d-11eb-892c-fcb9ac12306b.png)